schemas/content/1.0/schema.json

```diff
@@ -86,7 +86,9 @@
                 "audio/ogg",
                 "audio/webm",
                 "audio/x-wav",
-                "audio/wav"
+                "audio/wav",
+                "application/json",
+                "application/quiz"
```

Comment:
"application/quiz" - is this mimeType reviewed with DC. Please let us have an explicit approval from DC before adding any new mimeType to the default schema list.

Reply:
Removed the "application/quiz" mimeType.